pyqtgraph/metaarray/MetaArray.py

```diff
@@ -358,7 +358,7 @@ def asarray(self):
         else:
             return np.array(self._data)
             
-    def __array__(self):
+    def __array__(self, dtype=None):
```

Comment:
Shouldn't this do something like:
```
if dtype is not None:
    return self.asarray().astype(dtype)
else:
    return self.asarray()
```
 ?

Reply:
....ahh you're right; I'll revert the merge and make that change